File: qualifier-week-3/non graded assignments/PPA3 - Not Graded.c
// Prefix Code
#include <stdio.h>
int main() {
    double radius;
    scanf("%lf", &radius);
    // Write solution code below
    double volume;
    volume = (4.0 / 3.0) * 3.14 * radius * radius * radius;
    printf("%.2lf", volume);
    return 0;
}

File: qualifier-week-3/non graded assignments/PPA3 - Not Graded.rs
fn main() {
    let mut radius: f64 = 0.0;

    println!("Enter the Radius of the Sphere: ");
    let _ = scanf!("{:f64}", &mut radius);

    let volume = (4.0 / 3.0) * std::f64::consts::PI * radius.pow(3.0);

    println!("Volume of the Sphere: {:.2f}", volume);
}
